tools/coredump/testdata/amd64/elixir-bandit-phoenix-jason.json

```diff
@@ -0,0 +1,216 @@
+        "Jason.encode!/2+0 in lib/jason.ex:162",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.JS.to_iodata/1+0 in lib/phoenix_live_view/js.ex:153",
+        "Phoenix.HTML.build_attrs/1+0 in lib/phoenix_html.ex:238",
+        "Phoenix.HTML.build_attrs/1+0 in lib/phoenix_html.ex:238",
+        "Phoenix.HTML.build_attrs/1+0 in lib/phoenix_html.ex:199",
+        "Phoenix.HTML.attributes_escape/1+0 in lib/phoenix_html.ex:191",
+        "ElixirLoadAppWeb.CoreComponents.-flash (overridable 1)/1-fun-4-/3+0 in lib/elixir_load_app_web/components/core_components.ex:112",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.Rendered.to_iodata/1+0 in lib/phoenix_live_view/engine.ex:149",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.Rendered.to_iodata/3+0 in lib/phoenix_live_view/engine.ex:165",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.Rendered.to_iodata/3+0 in lib/phoenix_live_view/engine.ex:165",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.Rendered.to_iodata/3+0 in lib/phoenix_live_view/engine.ex:165",
+        "Phoenix.HTML.Safe.Phoenix.LiveView.Rendered.to_iodata/3+0 in lib/phoenix_live_view/engine.ex:165",
+        "Phoenix.Controller.-template_render_to_iodata/4-fun-0-/5+0 in lib/phoenix/controller.ex:1008",
+        "telemetry:span/3+0 in /home/greg/elixir_load_app/deps/telemetry/src/telemetry.erl:324",
+        "Phoenix.Controller.render_and_send/4+0 in lib/phoenix/controller.ex:974",
+        "ElixirLoadAppWeb.PageController.action/2+0 in lib/elixir_load_app_web/controllers/page_controller.ex:1",
+        "ElixirLoadAppWeb.PageController.phoenix_controller_pipeline/2+0 in lib/elixir_load_app_web/controllers/page_controller.ex:1",
+        "Phoenix.Router.__call__/5+0 in lib/phoenix/router.ex:484",
+        "ElixirLoadAppWeb.Endpoint.plug_builder_call/2+0 in lib/elixir_load_app_web/endpoint.ex:1",
+        "ElixirLoadAppWeb.Endpoint.call (overridable 3)/2+0 in deps/plug/lib/plug/debugger.ex:136",
+        "ElixirLoadAppWeb.Endpoint.call/2+0 in lib/elixir_load_app_web/endpoint.ex:1",
+        "Phoenix.Endpoint.SyncCodeReloadPlug.do_call/4+0 in lib/phoenix/endpoint/sync_code_reload_plug.ex:22",
+        "Bandit.Pipeline.call_plug!/2+0 in lib/bandit/pipeline.ex:127",
+        "Bandit.Pipeline.run/4+0 in lib/bandit/pipeline.ex:37",
+        "Bandit.HTTP1.Handler.handle_data/3+0 in lib/bandit/http1/handler.ex:12",
+        "Bandit.DelegatingHandler.handle_data/3+0 in lib/bandit/delegating_handler.ex:18",
+        "Bandit.DelegatingHandler.handle_info/2+0 in /home/greg/elixir_load_app/deps/thousand_island/lib/thousand_island/handler.ex:385",
+        "gen_server:try_handle_info/3+0 in gen_server.erl:2345",
+        "gen_server:handle_msg/6+0 in gen_server.erl:2433",
+        "proc_lib:init_p_do_apply/3+0 in proc_lib.erl:329"
+      ]
+    },
+    {
+      "lwp": 556614,
+      "frames": [
+        "libc.so.6+0x10e7db",
+        "beam.smp+0x220852",
+        "<unwinding aborted due to error native_no_pid_page_mapping>"
```

Comment:
Any idea what causes these?

Reply:
No, this is still a mystery to me that I'd really love to understand because I feel like it shouldn't be possible based on where that error string seems to be coming from, but here we are. It's very likely that it's due to me doing something wrong with the mappings, but that's a part of the code I really don't understand very well.